navit/vehicleprofile.c

```diff
@@ -198,7 +198,9 @@ vehicleprofile_update(struct vehicleprofile *this_)
 {
 	struct attr_iter *iter=vehicleprofile_attr_iter_new();
 	struct attr profile_option;
-	dbg(lvl_debug,"enter\n");
```

Comment:
Hmm, do we really want those messages on error level? Suppose to change them to debug before merge.